src/Components/Table/DjTable.tsx

```diff
@@ -164,23 +181,41 @@ export default class DjTable extends React.Component<
     let apiUrl =
       `${process.env.REACT_APP_DJSCIVIZ_BACKEND_PREFIX}` + this.props.route
 
-    if(!this.state.sorter.join(",").includes('ASC') && !this.state.sorter.join(",").includes('DESC')){
-      apiUrl = apiUrl + '?page=' + this.state.offset + '&limit=' + this.state.numberOfTuples;
-    } else { 
-      apiUrl = apiUrl + '?page=' + this.state.offset + '&limit=' + this.state.numberOfTuples + '&order=' + this.state.sorter.join(",");
+    if (
+      !this.state.sorter.join(',').includes('ASC') &&
+      !this.state.sorter.join(',').includes('DESC')
+    ) {
+      apiUrl =
+        apiUrl +
+        '?page=' +
+        this.state.offset +
+        '&limit=' +
+        this.state.numberOfTuples
```

Comment:
I'd normally recommend using [URLSearchParams](https://developer.mozilla.org/en-US/docs/Web/API/URLSearchParams#examples) since it is a bit more readable and safer.

```
const params = {foo: "hi there", bar: "100%" };
const queryString = new URLSearchParams(params).toString();
console.log(queryString);
```

Reply:
Done.